antidox/wikiwatcher.py

```diff
@@ -32,13 +74,21 @@ def watcher(event_source, wiki_filter, namespaces_filter, callback):
   """
   for event in event_source:
     if event.event == 'message' and event.data:
-      change = json.loads(event.data)
+      try:
+        change = json.loads(event.data)
+      except json.decoder.JSONDecodeError as err:
+        print("Error:", err)
+        pprint.pprint(event.data)
```

Comment:
If there are errors, we shouldn't continue with the rest of the loop. So in this `except` block after the pprint, we should also do `continue` to skip to the next change/event.